app/views/pages/home.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Startseite</h1>
```

Comment:
Fehlt hier die EditorConfig, wie in Folien der Vorlesung beschrieben? Jedenfalls fehlen bei diversen Files eine Zeile am Ende.